src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/AbstractFactory.php

```diff
@@ -67,7 +67,12 @@ public function addConfiguration(NodeDefinition $node)
         ;
 
         foreach (array_merge($this->options, $this->defaultSuccessHandlerOptions, $this->defaultFailureHandlerOptions) as $name => $default) {
-            if (\is_bool($default)) {
+            if ('require_previous_session' === $name) {
+                $builder
+                    ->booleanNode($name)
+                    ->setDeprecated('symfony/security-bundle', '6.4', 'Option "%node%" at "%path%" is deprecated. It does nothing and will be removed in version 7.0.')
```

Comment:
```suggestion
                    ->setDeprecated('symfony/security-bundle', '6.4', 'Option "%node%" at "%path%" is deprecated it will be removed in version 7.0. Setting it has no effect anymore.')
```

Reply:
I took the same message than `reset_on_message` on 6.1

But indeed, your suggestion is better !

It's applied